xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-ui/src/main/resources/XWiki/Like/LikeUIX.xml

```diff
@@ -142,7 +142,14 @@
       <cache>long</cache>
     </property>
     <property>
-      <code>require(['jquery', 'xwiki-meta'], function ($, xm) {
+      <code>define('xwiki-like-messages', {
+  prefix: 'like.button.title.',
+  keys: [
+    "unlike",
+    "like"
+  ]
+});
+require(['jquery', 'xwiki-meta', 'xwiki-l10n!xwiki-like-messages'], function ($, xm, l10n) {
```

Comment:
```suggestion
require(['jquery', 'xwiki-l10n!xwiki-like-messages', 'xwiki-meta'], function ($, l10n) {
```
Small detail but since  `xm` is not used, you can make it the last required of the array and remove the corresponding param.

Reply:
Forget this comment, I'd forgotten to unfold some part of the code...